docs/index.rst

```diff
@@ -158,6 +158,17 @@ Since version ``v0.17`` nested dataclasses are supported both in dataclasses and
 
 Dataclasses attributes can be populated by tuples, dictionaries or instances of that dataclass.
 
+post_init_post_parse
```

Comment:
"dataclass int hooks"

We don't currently have documentation on `__post_init__` so this is covering both functions.